server/interfaces/interfaces.go

```diff
@@ -572,11 +572,9 @@ type HealthChecker interface {
 
 // Locates all Xcode versions installed on the host system.
 type XcodeLocator interface {
-	// Returns the developer directory and SDKs for the given Xcode version.
-	DeveloperDirForVersion(version string) (string, error)
-
-	// Returns true if the given SDK path is present in the given Xcode version.
-	IsSDKPathPresentForVersion(sdkPath, version string) bool
+	// Finds the Xcode that most closely matches the given Xcode version.
```

Comment:
nit: could we specify in the comment what "closely matches" means -- what counts as a match versus an error?